appeals/web/package.json

```diff
@@ -13,6 +13,8 @@
     "clean": "rimraf .turbo node_modules dist",
     "clean:dev": "rimraf dist src/server/static/scripts src/server/static/styles",
     "dev": "cross-env NODE_ENV=local npm-run-all --silent clean:dev sass rollup --parallel --race watch:** nodemon",
+		"dev-permissions-co":   "cross-env HTTP_PORT=8080 HTTPS_PORT=8080 APP_HOSTNAME=localhost:8080 AUTH_REDIRECT_URI=https://localhost:8080/auth/redirect AUTH_DISABLED_GROUP_IDS=cc4133e5-2319-4762-8a7b-33413701210a NODE_ENV=local npm-run-all --silent clean:dev sass rollup --parallel --race watch:** nodemon",
```

Comment:
We probably don't want that `AUTH_DISABLED_GROUP_IDS=cc4133e5-2319-4762-8a7b-33413701210a` in here do we?

Reply:
ah, good catch! I'll remove the PR

Reply:
According to Ben it's not secret information